sql/log.cc

```diff
@@ -3951,8 +3951,9 @@ bool MYSQL_BIN_LOG::open(const char *log_name,
                                   (my_off_t)opt_binlog_gtid_index_span_min);
           if (!gtid_index)
             sql_print_information("Could not create GTID index for binlog "
-                                  "file '%s'. Accesses to this binlog file will "
-                                  "fallback to slower sequential scan.");
+                                  "file '%s'. Accesses to this binlog file "
+                                  "will fallback to slower sequential scan.",
+                                  log_file_name);
```

Comment:
I guessed this one and would like a confirmation.

Reply:
Yep, spot on. Good catch too.